server/models/robotJobModel.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const inputParameterSchema = new Schema({
+  name: String,
+  value: Schema.Types.Mixed,
+});
+
+// eslint-disable-next-line camelcase
+const Job_Schema = new Schema({
+  user_id: mongoose.Types.ObjectId,
+  robot_id: mongoose.Types.ObjectId,
+  status: String,
+  parameters: [inputParameterSchema],
+});
+
+const Job = mongoose.model('jobs', Job_Schema);
```

Comment:
MOngoose will add the s at the end by itself as far as my understanding goes, so it shouldn't be set.
```suggestion
const Job = mongoose.model('job', Job_Schema);
```